service/service.go

```diff
@@ -114,6 +142,29 @@ func NewService(cfg *Config) (*Service, error) {
 	return s, nil
 }
 
+func serializeProofMsg(r *round) ([]byte, error) {
+	poetProof, err := r.proof(false)
+	if err != nil {
+		return nil, fmt.Errorf("failed to get poet proof for round %d: %v", r.Id, err)
+	}
+	proofMessage := PoetProofMessage{
+		GossipPoetProof: GossipPoetProof{
+			MerkleProof: *r.nip,
+			Members:     r.challenges,
+			LeafCount:   uint64(1) << poetProof.N,
+		},
+		PoetId:    nil,
```

Comment:
why nil? and what about signing the msg?

Reply:
PoET identities aren't implemented yet and I don't think they should be part of this PR. If you disagree, lmk. (this should answer both your questions)